cmd/config/heal/heal.go

```diff
@@ -40,6 +48,10 @@ var (
 			Key:   Bitrot,
 			Value: config.EnableOff,
 		},
+		config.KV{
```

Comment:
It would also make sense to be able to specify the max amount of IO operations.

Reply:
We can do that too..

Reply:
added it